app/assets/stylesheets/expenses.css.scss

```diff
@@ -0,0 +1,12 @@
+#create_expense_link {
+  float: right;
+}
+
+#create_expense_link_text {
+  margin-top: 36px;
+}
+
+#expenses_page_heading {
```

Comment:
Avoid using id selectors<br>Selector `expenses_page_heading` should be written in lowercase with hyphens